lib/workers/repository/config-migration/pr/index.ts

```diff
@@ -35,8 +34,10 @@ export async function ensureConfigMigrationPr(
   });
   const filename = migratedConfigData.filename;
   logger.debug('Filling in config migration PR template');
-  let prTemplate = `Config migration needed, merge this PR to update your Renovate configuration file.\n\n`;
-  prTemplate += emojify(
+  let prBody = `The Renovate config in this repository needs migrating. Typically this is because one or more configuration options you are using have been renamed.
+
+  This PR does not need to be merged urgently, because Renovate will continue to migrate these fields internally each time it runs, but in future some of these fields may be fully deprecated and migrations removed, so it's best to apply such migrations when they are suggested.\n\n`;
```

Comment:
```suggestion
  You don't need to merge this PR right away, because Renovate will continue to migrate these fields internally each time it runs. But later some of these fields may be fully deprecated and the migrations removed. So it's a good idea to merge this migration PR soon. \n\n`;
```

What's the key point ? I guess it's something like:

> Please merge this PR sometimes in the next few weeks, so you're up to date with current terms, and don't get problems when the Renovate team deprecates the feature, and removes the migrations.

Reply:
I don't want to alarm people that "you must merge this now or Renovate won't keep running"

Reply:
I agree that we shouldn't scare our users. I'm not happy with the current wording though.

How about something like this:

> You do not need to merge this PR right away, because Renovate will continue to migrate these fields internally each time it runs. But later we may fully deprecate some of these fields and remove the migrations. We recommend you apply our migrations when they are suggested.

This text makes it clear who does what, and puts the reader into the picture:

- Use `we` to talk about us as Renovate developers:
  - We remove the fields and drop the migrations
  - We recommend the user merge the config migration PR
- Use `you` to talk about our user:
  - You don't need to merge this PR right away
  - You should apply our migrations